packages/core/src/element/click.ts

```diff
@@ -1,5 +1,6 @@
-import userEvent from '@testing-library/user-event'
+// @ts-ignore 123
```

Comment:
Should we prefer `@ts-expect-error` to allow for Deno to inform us when it isn't needed?